docs/configuring-socorro.rst

```diff
@@ -8,6 +8,19 @@ Socorro produces graphs and reports, most are updated once per day.
 You must enter information about your releases into Socorro in order
 for this to work, and this information must match the incoming crashes.
 
+Configure Elasticsearch
+-----------------------
+
+Many features of Socorro depend on Elasticsearch, and to work you will first
```

Comment:
Mmm this is all explained if you follow the link. I don't really want to repeat things too much. Do you have a suggestion of a good catch phrase to sum it up?